code/game/objects/structures/__structure.dm

```diff
@@ -134,8 +134,9 @@
 		last_damage_message = 0.75
 
 /obj/structure/physically_destroyed(var/skip_qdel)
-	. = ..(TRUE)
-	dismantle()
+	if(!(. = ..(TRUE)))
```

Comment:
Why?
Adding a `if(.)` before the `dismantle()` would have done the same job.

Reply:
Just to annoy lohikar tbh :P
But, yeah, not really any reasons. It just fits on one line.
Looking at it now, I'm not even really using the return value anyways, so I probably could just do if(!..(TRUE))